source/_components/solaredge_local.markdown

```diff
@@ -0,0 +1,109 @@
+---
+layout: page
+title: "SolarEdge Local Sensor"
+description: "Instructions on how to integrate SolarEdge sensor within Home Assistant via Local API."
+date: 2018-12-04 14:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: solaredge.png
+ha_category: Sensor
+ha_release: 0.94
+ha_iot_class: Local Polling
+redirect_from:
+ - /components/sensor.solaredge_local/
+---
+
+The `solaredge_local` platform uses the local API available on some SolarEdge Inveters to allow you to get details from your SolarEdge solar power setup and integrate these in your Home Assistant installation.
+
+
+Only specific models support the local API.  The local API is available on the SExxxxH-US models with SetApp as well as European three phase inverters SEXXK-XXXTXBXX4 models with SetApp like SE3K-E10K, SE12.5K-SE27.6K and SE33.3K. Please check carefully the datasheets if in the section "Additional Features", sub-section "Inverter Commissioning" is present the following line "With the SetApp mobile application using built-in Wi-Fi access point for local connection".
+
+You can check by finding the IP address of your inverter and visiting it in a browser. If it supports the local API, you'll see the SolarEdge logo and a "Commissioning" menu.
+
+<p class='note'>
+If your inveter does not support the local API, you can use the [cloud based version](/components/solaredge/) instead
+</p>
+
+## {% linkable_title Configuration %}
+
+To use the SolarEdge sensors in your installation, add the following to your configuration.yaml file:
+
+{% raw %}
+```yaml
+# Example configuration.yaml entry
+sensor:
+  platform: solaredge_local
+  ip_address: 192.168.1.130
+  monitored_conditions:
```

Comment:
Is this required?

Reply:
no, it will default to `current_power` only.  I followed the convention of the existing solar edge component.  I would think most users would want to specify though.  Should I remove it?